recipes/jupyterlab_genv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab_genv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_genv-{{ version }}.tar.gz
+  sha256: c850e8670f10e8c6926de1fce8b85de17ffdb69a6cccc3ac8cdc38fd6c4954f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - hatchling >=1.3.1
+    - jupyterlab ~=3.1
+    - hatch-jupyter-builder >=0.5
```

Comment:
This is not listed upstream. Is it required?

Reply:
yes it didn't work without it because of this https://github.com/run-ai/jupyterlab_genv/blob/main/pyproject.toml#L2

Reply:
Yes, but `hatch-jupyter-builder` is not listed there. It's fine now, but ideally upstream and here are the same so it should be listed upstream as well.